### ISSUE TITLE
Unkown size file transfert not working 

### ISSUE THREAD

**Issue by z3bra** [evilcorpltd/aTox#1063]
When a contact sends a file of unknown size (with a [size of "`UINT64_MAX`", as per the doc](https://github.com/TokTok/c-toxcore/blob/master/toxcore/tox.h#L2188,L2190), the transfert is immediately canceled on aTox side, without sending `TOX_FILE_CONTROL_CANCEL`, thus leaving the sender in pending state.

On aTox side, the filesize is seen a "-1 byte(s)", which might be the root of the issue.

**Comment by robinlinden** [comment]
Yeah, streaming file transfers aren't yet supported. We should make this more clear for now, and also implement support for them in the future.
